src/client/chat.js

```diff
@@ -102,7 +102,7 @@ module.exports = function (client, options) {
     }
 
     // Chain integrity remains even if message is considered unverified due to expiry
-    const tsDelta = Date.now() - packet.timestamp
+    const tsDelta = BigInt(Date.now()) - packet.timestamp
```

Comment:
This doesn't seem correct, will the `tsDelta > messageExpireTime` work ? The latter is a JS Number. packet.timestamp could instead be handled as a Number

Reply:
BigInt and Number can be compared as usual